packages/adapter-pg-lite/README.md

```diff
@@ -0,0 +1,38 @@
+# @prisma/adapter-pg-lite
+
+This package contains the driver adapter for Prisma ORM that enables usage of the [`PGlite`](https://github.com/electric-sql/pglite) database driver for PostgreSQL running in WebAssembly. You can learn more in the [documentation](https://pris.ly/d/adapter-pg-lite).
+
+`PGlite` is a lightweight, WASM-based PostgreSQL build that allows running PostgreSQL directly in the browser, Node.js, and Bun environments without any external dependencies. It is only 2.6mb gzipped and supports both ephemeral in-memory databases and persistent storage.
+
+> **Note:**: Support for the `PGlite` driver is available from Prisma versions [5.4.2](https://github.com/prisma/prisma/releases/tag/5.4.2) and later.
+
+## Usage
+
+This section explains how you can use it with Prisma ORM and the `@prisma/adapter-pg-lite` driver adapter. Be sure that the `DATABASE_URL` environment variable is set to your PostgreSQL connection string (e.g., in a `.env` file) or use a direct in-memory connection.
+
+### 1. Enable the `driverAdapters` Preview feature flag
+
+Since driver adapters are currently in [Preview](/orm/more/releases#preview), you need to enable its feature flag on the `datasource` block in your Prisma schema:
+
+```prisma
+// schema.prisma
+generator client {
+  provider        = "prisma-client-js"
+  previewFeatures = ["driverAdapters"]
+}
+
+datasource db {
+  provider = "postgresql"
```

Comment:
Should this be `pglite` instead?
```suggestion
  provider = "pglite"
```

Reply:
Interesting. I went off of the new provider added here

https://github.com/BLamy/prisma/blob/bd007e4b5f2cc2063d828c2325642de4d3388f70/packages/driver-adapter-utils/src/types.ts#L86
`readonly provider: 'mysql' | 'postgres' | 'sqlite' | 'pglite'`

Reply:
I’d have to check with the Prisma team by I thought of it as “flavor” and “client” — meaning this flavor of SQL and the features it supports is “postgres” (for example arrrays and enums are supported), but the client used to make queries etc is the adapter “pglite”. But still learning.

Reply:
Yeah I was a little confused by this. I think what I want is that I want postgres to be used on both prod and dev. But if NODE_ENV=test I want it to be pglite.